packages/notifi-graphql/lib/gql/queries/stateChanged.gql.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from 'graphql-request';
+
+export const stateChanged = gql`
+subscription stateChanged {
+  stateChanged {
```

Comment:
Not sure if this endpoint available? I cannot find it either in prod or env. 
So it might cause build fail.